src/main/java/org/oscii/api/LexiconProtocol.java

```diff
@@ -42,6 +52,9 @@ public Response respond(Request request) {
         if (request.example) addExamples(request, response);
         if (request.extend) addExtensions(request, response);
         if (request.synonym) addSynonyms(request, response);
+        if (request.wordvec) addWordVector(request, response);
```

Comment:
you're right! lookup is supported if you formulate the query accordingly (e.g. `San_Francisco`), but ranking of concordance entries does not take those into account; this would require extraction of bigrams/trigrams in the context and the sentence examples (not sure if the trigrams are part of the model, need to check) and score them instead of the unigram matches.